docs/1.18.x/setup/configuration/buildscript.md

```diff
@@ -0,0 +1,41 @@
+# Setup - Configuration
+
+By buildscript, we refer to the `build.gradle` file of your project.
```

Comment:
Add more information here. What does it do? What can it do? When is it used? What is gradle?